etc/PackageConfigurations/eos_package_description.json

```diff
@@ -35,7 +35,8 @@
         {"src":"Assets/Plugins/Source/Core/*.asmdef.meta", "dest": "Runtime/Core/"},
         {"src":"Assets/Plugins/Source/Core/*.cs", "dest": "Runtime/Core/"},
         {"src":"Assets/Plugins/Source/Core/*.cs.meta", "dest": "Runtime/Core/"},
-        
+        {"src":"Assets/Plugins/Source/Core/Utility/*.cs", "dest": "Runtime/Core/Utility/"},
+        {"src":"Assets/Plugins/Source/Core/Utility/*.cs.meta", "dest": "Runtime/Core/Utility/"},
```

Comment:
The scripts in `Source/Editor/Windows` are missing, a believe those are editor windows for users.

Reply:
Excellent catch, thanks Arthur. I've since fixed the issue.

Reply:
For some reason, GitHub is not marking this file as being outdated 🤔